jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/LifeCycleBaseTest.java

```diff
@@ -2388,7 +2390,9 @@ public void beforeTaskStartedEvent(TaskEvent event) {
 
             @Override
             public void beforeTaskCompletedEvent(TaskEvent event) {
-                assertNull(event.getTask().getTaskData().getTaskInputVariables());                
+                assertNotNull(event.getTask().getTaskData().getTaskInputVariables());
+                assertEquals(1, event.getTask().getTaskData().getTaskInputVariables().size());
+                assertTrue(event.getTask().getTaskData().getTaskInputVariables().containsKey("input"));                
```

Comment:
OK, I think you have a point there. But I think I don't understand the last sentence. So this listener is not guaranteed to be triggered so there can be situations which data are not loaded in?